pkg/kubelet/dockertools/docker.go

```diff
@@ -1049,3 +1049,21 @@ func RunContainer(client DockerInterface, container *api.Container, pod *api.Pod
 	}
 	return dockerContainer.ID, nil
 }
+
+// KillContainer kills a docker container with the given containerID.
+// TODO(yifan): To use strong type for the containerID.
+func KillContainer(client DockerInterface, containerID string,
```

Comment:
@vmarmol OK, I think container runtime interface should not hide the implicit killing/restart. Probably it's better to add some code to kubelet to handle different container runtime.